streaming/text/convert/enwiki/mds/pick_eval_samples.py

```diff
@@ -7,17 +7,46 @@
 
 def parse_args():
     args = ArgumentParser()
-    args.add_argument('--input_dir', type=str, required=True)
-    args.add_argument('--output_dir', type=str, required=True)
-    args.add_argument('--compression', type=str, default='zstd:16')
-    args.add_argument('--hashes', type=str, default='sha1,xxh3_64')
-    args.add_argument('--size_limit', type=int, default=1 << 26)
-    args.add_argument('--num_examples_to_pick', type=int, default=10000)
+    args.add_argument(
+        '--in_root',
+        type=str,
+        required=True,
+        help='Local directory path of the input raw dataset',
+    )
+    args.add_argument(
+        '--out_root',
+        type=str,
+        required=True,
+        help='Directory path to store the output MDS shard files',
+    )
+    args.add_argument(
+        '--compression',
+        type=str,
+        default='zstd:16',
+        help='Compression algorithm to use. Default: zstd:16',
+    )
+    args.add_argument(
+        '--hashes',
+        type=str,
+        default='sha1,xxh64',
```

Comment:
you changed the default?

Reply:
Nice catch. Fixed in https://github.com/mosaicml/streaming/pull/171/commits/b6539a43b3035c50d86be6554786296bdc1d6df4.